rust/worker/src/blockstore/positional_posting_list_value.rs

```diff
@@ -89,6 +89,18 @@ impl PositionalPostingListBuilder {
         Ok(())
     }
 
+    pub(crate) fn delete_doc_id(&mut self, doc_id: i32) -> Result<(), Box<dyn ChromaError>> {
```

Comment:
I think @HammadB was recommending to NOT use `Box<dyn ChromaError>` since it makes 'match'ing against it very hard. We should have concrete enums here